docs/index.asciidoc

```diff
@@ -56,6 +66,28 @@ input plugins.
 
 The maximum packet size to read from the network
 
+[id="plugins-{type}s-{plugin}-ecs_compatibility"]
+===== `ecs_compatibility`
+
+* Value type is <<string,string>>
+* Supported values are:
+** `disabled`: unstructured connection metadata added at root level
+** `v1`: structured connection metadata added under ECS compliant namespaces
+* Default value depends on which version of Logstash is running:
+** When Logstash provides a `pipeline.ecs_compatibility` setting, its value is used as the default
+** Otherwise, the default value is `disabled`.
+
+Controls this plugin's compatibility with the https://www.elastic.co/guide/en/ecs/current/index.html[Elastic Common Schema (ECS)].
+The value of this setting affects the placement of a TCP connection's metadata on events:
+
+.Metadata Location by `ecs_compatibility` value
+[cols="<l,<l,e,<e"]
```

Comment:
I would like to keep this table, it's something standard we try to use on all ECS adapatations, to that the user can have a complete summary of all the ECS remmaping applied. In this case with only one fields it may be redundant.